docs/new-curriculum/js-core-1/week-1/notes.md

```diff
@@ -0,0 +1,349 @@
+However, we can use computers to evaluate expressions. We can use NodeJS, a JavaScript runtime, an application that runs JavaScript programs.
+
+Node can be used in a number of different ways; however, one particular mode is the REPL. The Node REPL enables us to input JavaScript expressions which are then evaluated by NodeJS and printed to the terminal. The REPL will print out the result of this evaluation.
+
+:::note
+
+ℹ️ Definition: REPL is a special type of program that stands for
+
+**Read**  
+**Evaluate**  
+**Print**  
+**Loop**
+
+:::
+
+We can type each of the following expressions into the REPL and then press enter to check the result.
+
+```bash
+10 + 32
+```
+
+```bash
+32 / 10
+```
+
+```bash
+3 ** 4
+```
+
+### Saving expressions
+
+Often in programming we may want to reuse a particular expression in our program. In this case, we need a way to refer back to a particular expression. Let’s consider the following string:
+
+```js
+"Hello there";
+```
+
+Suppose we want to reuse this string in several different places. For example, we may want to create different greetings for different users, like:
+
+```bash
+"Hello there, Alicia";
+```
+
+```bash
+"Hello there, Barny"
+```
+
+We can use a **variable** to store this string value so it can be referenced and used again.
+
+:::info
+
+A **variable** is a label for a piece of data. We can assign a piece of data to a particular label and then refer back to this label.
+
+:::
+
+We can create a variable in our program by writing a **variable declaration**, in the following way:
+
+```js title="variable declaration"
+const greeting = "Hello there";
+```
+
+We can break down the different syntactic elements of this variable declaration:
+
+- `const` is a keyword used to indicate we're creating a variable.
+- `greeting` is the variable name - like the name of the label for our piece of data.
+- `=` this is the assignment operator. It means assign to the label `greeting` the expression on the right hand side.
+- `"Hello there"` - this is the value we're assigning to the label `greeting`.
+
+We can type this variable declaration into the REPL:
+
+![greeting-variable-declaration](./assets/greeting-variable-declaration.png)
+
+Now we can refer to the label `greeting` again in the REPL:
+
+![reference-greeting](./assets/reference-greeting.png)
+
+Now we have the `greeting` variable stored in memory we can reuse it to build more expressions:
+
+![use-greeting-variable](./assets/use-greeting-variable.png)
+
+:::note
+
+In the example above, we're using backticks to create a template literal. With template literals, we can insert expressions into strings to produce new strings. Any time we want to reference a variable inside a template literal we use a dollar sign and a set of curly braces. The variable name itself is then placed inside the curly braces. The value this variable holds is then placed inside the string.
+
+:::
+
+### Reusing instructions
```

Comment:
same as for variables, I'm not sure that there's a point in introducing functions until we can motivate what happens when you don't have reusable instrucitons

Reply:
I think this section does give some motivation on why functions are useful: there's the discussion of rounding numbers repeatedly.

Perhaps reframing this as a question for trainees would make it more obvious?